libs/tendermint/mempool/reactor.go

```diff
@@ -106,9 +114,14 @@ func newMempoolIDs() *mempoolIDs {
 // NewReactor returns a new Reactor with the given config and mempool.
 func NewReactor(config *cfg.MempoolConfig, mempool *CListMempool) *Reactor {
 	memR := &Reactor{
-		config:  config,
-		mempool: mempool,
-		ids:     newMempoolIDs(),
+		config:           config,
+		mempool:          mempool,
+		ids:              newMempoolIDs(),
+		nodeKeyWhitelist: make(map[string]struct{}),
```

Comment:
Maybe in the future we could use the Value directly. better for []byte or string ?